app/views/qsets/show.html.erb

```diff
@@ -1,101 +1,36 @@
-                  <div class="question-voting">
-                    <div class="glyphicon glyphicon-triangle-top upvote-button" data-vote="upvote" data-vote-qid="<%= question.id %>"></div>
-                    <div class="glyphicon glyphicon-triangle-bottom downvote-button" data-vote="downvote" data-vote-qid="<%= question.id %>"></div>
-                  </div>
-                </div>
-                <div class="col-xs-8 col-sm-9">
-                  <div class="question-content">
-                    <div class="question-content-inner">
-                      <span>
-                        <%= link_to question_path(question) do%>
-                        <%= question.text %>
-                        <% end %>
-                      </span>
-                    </div>
-                    <br/>
-                    <div class="pull-right">
-                        <!-- Creates the edit and delete options for each question -->
-                        <% if can? :manage, question %>
-
-                          <%= link_to edit_question_path(question), class:'btn shortcut-button-link' do%>
-                            <span class="glyphicon glyphicon-edit shortcut-button-glyphicon"></span>
-                          <% end %>
-
-                          <%= link_to question_path(question), method: :delete, class:'btn shortcut-button-link' do%>
-                            <span class="glyphicon glyphicon-trash shortcut-button-glyphicon"></span>
-                          <% end %>
-
-                        <% end %>
-                    </div>
-                  </div>
-                </div>
-              </div>
-              <div class="col-xs-3">
-                <div class="col-sm-5">
-                  <%= link_to question.user.full_name, user_path(question.user.id) %>
-                </div>
-                <div class="col-sm-7">
-                    <a data-toggle="modal"
-                      href="#"
-                      data-target="#question_display_Modal"
-                      data-answer="<%= question.answers[0].text %>"
-                      data-qid="<%= question.id %>"
-                      data-question="<%= question.text %>"
-                      class="btn btn-primary">
-                        Quiz Me
-                    </a>
-                </div>
-              </div>
-            </div>
-          </h4>
-        </div>
-      </div> <!-- .panel panel-default -->
-      </div> <!-- .panel-group -->
-    <% end %>
+      <%= render "question_list" %>
   </div> <!-- .question-list -->
   <br>
-</div> <!-- .container -->
-
-<%= render "modal_question" %>
-<%= render "modal_social" %>
-<% if can? :manage, Qset %>
-  <%= render "modal_edit_qset" %>
-  <%= render "modal_new_qset" %>
-<% end %>
 
-<script>
-  $(document).ready(function(){
-    initQsets();
-    initQuestionFilter();
-    initVote();
-  });
-</script>
+  <%= render "modal_question" %>
```

Comment:
I think the modals and `<script>` section can/should still be outside of this outer `<div>`. They are hidden, and scripts are not display elements, so I don't think they belong in the `.container` `div` element.